### DISCUSSION TITLE
Crash in skin editor after deleting files

### DISCUSSION THREAD

**Discussion by cakestheking** [ppy/osu#20394] [Q&A]
I was digging in my PC files to see if I could free up space, I opened the osu folders, saw a folder named ''files'' upon opening it and seeing its contents I mistook them for cache files and deleted them. Turns out, those files were beatmaps and other files. I eventually got all of my beatmaps and skins back, but the skin editor doesn't work now, it just crashes the game if I try to open it. what do I do?

Log files:
[network.log](https://github.com/ppy/osu/files/9622826/network.log)
[performance.log](https://github.com/ppy/osu/files/9622827/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9622828/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/9622829/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9622830/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9622831/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9622832/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9622833/updater.log)
[database.log](https://github.com/ppy/osu/files/9622834/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9622835/legacy-ipc.log)

**Comment by peppy** [comment]
Could you please [attach log files](https://github.com/ppy/osu/discussions/15188) so we can better understand what's going wrong?
